discord-groups/style.css

```diff
@@ -132,18 +141,18 @@
 */
 
 .loader {
-  border: 8px solid var(--color-loader-border);
-  border-top: 8px solid var(--color-loader-border-top);
+  border: 0.6rem solid var(--color-loader-border);
+  border-top: 0.6rem solid var(--color-loader-border-top);
```

Comment:
Nit: @chaithraramesh31 use static unit on border
because we want border size always fixed
Does it make sense to use rem here

Correct me If I am wrong??

Reply:
@Pratiyushkumar bhai can you please reply on this comment

Reply:
> Nit: @chaithraramesh31 use static unit on border
> because we want border size always fixed
> Does it make sense to use rem here
> 
> Correct me If I am wrong??
> 

No any values are not fixed, it depends on the client machine or display resolution i think.

Reply:
> It was in px before, but pratiyush told me to change all the measurements to rem. Should I change it to px?

It is good practice to use REM but try to use PX if it is less than 8px(not the rule but it is good to maintain).

Reply:
@chaithraramesh31   you can replace the border size rem with px only and 
@sahsisunny What I have understood is you should always depend on one unit only and places where size is fixed like the border which does not depend on screen size can be in px because px is a unit whose measurement can vary from screen size to screen size
rest let's discuss it with Ankush over this instead of debating it because I guess everyone has some good points and I see from 3-4 PR's we have a lot of discussion over this but have not been able to reach a conclusion.

cc- @shubhamsigdar1